clash-ghc/src-bin-common/Clash/GHCi/Common.hs

```diff
@@ -114,7 +115,7 @@ checkClashDynamic dflags = do
   let isStatic = case lookup "GHC Dynamic" (DynFlags.compilerInfo dflags) of
         Just "YES" -> False
         _          -> True
-  when isStatic
```

Comment:
This seems wrong... shouldn't we check for the os (`System.Info.os`) on which we run, not the one we build with.

Reply:
Especially if we ever to decide to crosscompile.

Reply:
Sorry for merging. Yes you're right.